bool isStarted
---
float64 direction
